services/QuillLMS/client/app/bundles/Staff/components/comprehension/configureRules/ruleViewForm.tsx

```diff
@@ -0,0 +1,261 @@
+      ruleFeedbacks,
+      ruleOptimal,
+      rulePrompts,
+      rulePromptIds: [promptId],
+      rulesCount,
+      ruleType,
+      setErrors,
+      submitRule,
+      universalRulesCount
+    }).then(() => {
+      setIsLoading(false);
+    });
+  }
+
+  const returnLinkRuleType = getReturnLinkRuleType(ruleType);
+  const returnLinkLabel = getReturnLinkLabel(ruleType);
+
+  function handleDeleteRule() {
+    let ruleId = id.toString();
+    if(!ruleId) {
+      ruleId = location.state.rule.id;
+    }
+    deleteRule(ruleId).then((response) => {
+      toggleShowDeleteRuleModal();
+      // update ruleSets cache to remove delete ruleSet
+      queryCache.refetchQueries(`rules-${activityId}`);
+      history.push(`/activities/${activityId}/${returnLinkRuleType}`);
+    });
+  }
+
+  const cancelLink = (<Link to={`/activities/${activityId}/${returnLinkRuleType}`}>Cancel</Link>);
+  const autoMLParams = {
+    label: 'Descriptive Label',
+    notes: 'Label Notes',
+    layerFeedback: 'First Layer Feedback'
+  }
+
+  if(isLoading) {
+    return(
+      <div className="loading-spinner-container">
+        <Spinner />
+      </div>
+    );
+  }
+
+  const formErrorsPresent = !!Object.keys(errors).length;
+  const requestErrorsPresent = !!(requestErrors && requestErrors.length);
+  const showErrorsContainer = formErrorsPresent || requestErrorsPresent;
+
+  return(
+    <div className="rule-form-container">
+      {showDeleteRuleModal && renderDeleteRuleModal()}
+      <section className="semantic-rule-form-header">
+        <Link className="return-link" to={`/activities/${activityId}/${returnLinkRuleType}`}>{returnLinkLabel}</Link>
+        <button className="quill-button fun primary contained" id="rule-delete-button" onClick={toggleShowDeleteRuleModal} type="button">
+          Delete
+        </button>
+      </section>
+      <form className="semantic-rule-form">
+        <RuleGenericAttributes
+          autoMLParams={autoMLParams}
+          concepts={conceptsData ? conceptsData.concepts : []}
+          errors={errors}
+          isAutoML={true}
+          isUniversal={isUniversal}
+          ruleConceptUID={ruleConceptUID}
+          ruleDescription={ruleDescription}
+          ruleID={id}
+          ruleName={ruleName}
+          ruleOptimal={ruleOptimal}
+          ruleType={ruleType}
+          ruleTypeDisabled={ruleTypeDisabled}
+          ruleUID={uid}
+          setRuleConceptUID={setRuleConceptUID}
+          setRuleDescription={setRuleDescription}
+          setRuleName={setRuleName}
+          setRuleOptimal={setRuleOptimal}
+          setRuleType={setRuleType}
+        />
+        {isSemantic && <RuleSemanticAttributes
+          errors={errors}
+          ruleLabelName={ruleLabelName}
+          ruleLabelNameDisabled={!!(label && label.name)}
+          ruleLabelStatus={ruleLabelStatus}
+          setRuleLabelName={setRuleLabelName}
+        />}
+        {ruleType && ruleType.value === 'plagiarism' && <RulePlagiarismAttributes
```

Comment:
we should probably have and be using constants for things like "plagiarism"

Reply:
good call, I'll add it as a const